### DISCUSSION TITLE
Quarkus + AWS Parameter Store

### DISCUSSION THREAD

**Discussion by andrefurtado11135** [quarkusio/quarkus#25319] [Q&A]
Hello!

I have some Spring applications and I would like to migrate these applications to Quarkus, but the configuration parameters of these applications are stored in the AWS Parameter Store and we replace them in the yaml file using the **spring-cloud-starter-aws-parameter-store-config** dependency and that allows us to overwrite the values directly into the yaml file. I find the Quarkus-SSM dependency to be close of what I need, but in the tutorials the parameters are being retrieved programatically and I would like to know if there is a way for these parameters to be retrieved and overwrite the values in the yaml file just like in Spring.

Thanks!

**Comment by petrucristescu** [comment]
Was anyone able to achieve this somehow?
I need to be able to retrieve a password for an ActiveMQ broker which is stored in AWS parameter store dynamically for every environment.

**Reply by andrefurtado11135** [reply]
Hey, unfortunately not. I would advise you to use kubernetes secrets to store the values and retrieve them, and to not use the AWS parameter store. At least this was the solution I found that suited me the best, not sure if that is possible in your case.
